runtime/dom-particle.js

```diff
@@ -109,12 +109,27 @@ export class DomParticle extends XenStateMixin(Particle) {
   async setHandles(handles) {
     this.handles = handles;
     let config = this.config;
-    this.when([new HandleChanges(handles, config.handles, 'change')], async () => {
-      await this._handlesToProps(handles, config);
-    });
+
+    // this.config isn't a static property; not sure how important it is to avoid re-generating
+    // it in the onHandle* functions
+    this._configSnapshot = config;
+    this._modelCount = this._configSnapshot.handles.length;
```

Comment:
I think this should use a set of the handle names to check the number of sync calls, rather than a countdown. It'll be easier to merge this PR as-is and I'll follow up with another for the set change.

Reply:
Roger that, also I agree (about using a set).